pkg/controllers/user/pipeline/engine/jenkins/engine.go

```diff
@@ -131,7 +132,7 @@ func (j *Engine) prepareRegistryCredential(pipeline *v3.Pipeline, stage int, ste
 	publishImageStep := pipeline.Spec.Stages[stage].Steps[step]
 	registry, _, _ := utils.SplitImageTag(publishImageStep.PublishImageConfig.Tag)
 
-	secrets, err := j.SecretLister.List(pipeline.Namespace, labels.Everything())
+	secrets, err := j.ManagementSecretLister.List(pipeline.Namespace, labels.Everything())
```

Comment:
The pipeline namespace is in cluster side, right? then we should use secretLister here?

Reply:
pipelines are the data store in the management plane and belong to a project so the `ManagementSecretLister ` is used here